### ISSUE TITLE
Incorrect (out of date?) Octopus Go rates

### ISSUE THREAD

**Issue by reedy** [emoncms/app#172]
Looking at the Octopus Agile app, the Go rates (for Yorkshire) seem rather out of date, using 5p/kWh and 16.67kW/h.

My fixes from March 2022 are:
<img width="758" alt="Screenshot 2022-08-26 at 00 56 55" src="https://user-images.githubusercontent.com/67615/186788984-96988e4a-e09e-490f-870e-eb457854c876.png">

Which are obviously quite a bit different, and therefore make it hard to compare to whether Agile makes sense...

**Comment by glynhudson** [comment]
Daytime rates are 43.29p/kWh in North Wales :-( 

The issue is some people are still on the old tariff, it's probably easier to let people input their own rates. It would also be good to make the off-peak period configurable, I'm on Go Faster, so my off-peak period is at a different time.

**Comment by reedy** [comment]
Yeah, I would concur allowing people to select their off peak time and enter their rates is probably the best way forward

**Comment by reedy** [comment]
Looks like @TrystanLea beat us to it c2d5d971a8e4a74b76e8a18d97aca5bd73daa9d4
